.github/workflows/gh-ci.yaml

```diff
@@ -30,11 +30,14 @@ env:
   HTML_DIR: "doc/build/html"
 
 jobs:
-  build_docs:
-    runs-on: ubuntu-latest
-    env:
-      CYTHON_TRACE_NOGIL: 1
-      MPLBACKEND: agg
+  tests:
```

Comment:
please keep the old name - this is a build docs CI run

Reply:
Fixed.